src/ui/page/accessibility-statement.html

```diff
@@ -20,7 +20,7 @@ <h1 class="heading-xlarge">Accessibility statement</h1>
                 <h2 class="heading-medium">How accessible this website is</h2>
                 <p>We know some parts of this website are not fully accessible:</p>
```

Comment:
Should this line be removed now that everything is good?

Reply:
Now it is decided not to include this ticket in the release. So following two new PRs are raised to remove these changes from the idt-develop branch and idt-release-1.0 branch.
https://github.com/Civil-Service-Human-Resources/lpg-services/pull/712/files
https://github.com/Civil-Service-Human-Resources/lpg-services/pull/713/files